src/Apps/Order/__tests__/OrderApp.test.tsx

```diff
@@ -48,17 +48,42 @@ describe("OrderApp routing redirects", () => {
     expect(redirect).toBe(undefined)
   })
 
-  it("redirects to the status route if the order is not pending", async () => {
+  it("redirects to the artwork page if the order is abandoned", async () => {
```

Comment:
Can we add back a test that fits the original description? 

Maybe one that is in a `SUBMITTED` state that does redirect to `status`?